browser/components/CodeEditor.js

```diff
@@ -94,8 +96,26 @@ export default class CodeEditor extends React.Component {
 
   componentDidMount () {
     const { rulers, enableRulers } = this.props
+    const emptyChars = /\t|\s|\r|\n/
+    if (!fs.existsSync(consts.SNIPPET_FILE)) {
+      const defaultSnippets = [
```

Comment:
That are quite a lot 'default' snippets not everybody needs. Nevertheless they will be inserted by default... not sure if that's a good idea

Reply:
You're right :smile: I think leaving just 1 default snippet is enough, will fix it soon